README.md

```diff
@@ -66,6 +66,14 @@ $ make quake
 
 The usage and limitations of Doom and Quake demo are listed in [docs/demo.md](docs/demo.md).
 
+### Docker image
+
+The image containing all the necessary tools for development and testing can be built by executing `docker build -t rv32emu_ubuntu .`. 
+
+This image works for both x86 and aarch64 (Apple's M1 chip) machines. Note that the image building time will be long since it will perform builds of the compiler toolchain and the reference emulator from the source code. Also, it will take up about 16GB of disk space.
```

Comment:
Simply refer to "toolchain" instead of "compiler toolchain."

Reply:
Fixed